metrics/grafana/tikv_details.dashboard.py

```diff
@@ -10307,6 +10308,235 @@ def ResourceControl() -> RowPanel:
     return layout.row_panel
 
 
+def TikvConfig() -> RowPanel:
+    layout = Layout(title="Config")
+
+    layout.row(
+        [
+            table_panel(
+                title="RocksDB DB Config",
+                description="TiKV RocksDB DB Configuration",
+                targets=[
+                    target(
+                        expr=expr_simple(
+                            "tikv_config_rocksdb_db",
+                            label_selectors=[
+                                'k8s_cluster="$k8s_cluster"',
+                                'tidb_cluster="$tidb_cluster"',
+                                'instance=~"$instance"',
+                            ],
+                        ),
+                        legend_format="",
+                    ),
+                ],
+                overrides=[
+                    {
+                        "matcher": {"id": "byName", "options": "Field"},
+                        "properties": [
+                            {"id": "displayName", "value": "Option"},
+                            {"id": "custom.align", "value": None},
+                        ],
+                    },
+                    {
+                        "matcher": {"id": "byName", "options": "Last (not null)"},
+                        "properties": [{"id": "displayName", "value": "Value"}],
+                    },
+                ],
+                time_from="1s",
+                transformations=[
+                    {
+                        "id": "organize",
+                        "options": {
+                            "excludeByName": {
+                                "Time": True,
+                                "__name__": True,
+                                "job": True,
+                            },
+                            "indexByName": {},
+                            "renameByName": {
+                                "Time": "",
+                                "Value #A": "Value",
+                                "name": "Option",
+                                "job": "",
+                            },
+                        },
+                    }
+                ],
```

Comment:
Looks like all table panels share the same `overrides`, `time_from` and `transformations`, can them be default settings?